src/application/services/axisValInputHandler/axisValInputHandler.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  AxisKey,
+  AxisValFormat,
+  AxisValuesInput,
+  AxisValuesInputValidationStatus,
+} from '@/@types/types'
+import { AxisValInputHandlerInterface } from './axisValInputHandlerInterface'
+import { MathUtils } from '@/application/utils/MathUtils'
+import { AxisSetRepositoryInterface } from '@/domain/repositories/axisSetRepository/axisSetRepositoryInterface'
+
+export class AxisValInputHandler implements AxisValInputHandlerInterface {
+  axisSetRepository: AxisSetRepositoryInterface
+  inputValues: Record<number, AxisValuesInput> = {}
+  validationStatus: Record<number, AxisValuesInputValidationStatus> = {}
+
+  constructor(axisSetRepository: AxisSetRepositoryInterface) {
+    this.axisSetRepository = axisSetRepository
+  }
+
+  getAxisSetInputValues(axisSetId: number): AxisValuesInput {
+    return (
+      this.inputValues[axisSetId] || {
+        x1: '0',
+        x2: '0',
+        y1: '0',
+        y2: '0',
+      }
+    )
+  }
+
+  getInputValueFormat(inputVal: string): AxisValFormat {
+    if (MathUtils.isScientificNotation(inputVal)) {
+      return 'scientificNotation'
+    }
+
+    if (MathUtils.isPowerNotation(inputVal)) {
+      return 'powerNotation'
+    }
+
+    if (MathUtils.isConvertibleToDecimal(inputVal)) {
+      return 'decimal'
+    }
+
+    return 'invalidFormat'
+  }
+
+  convertAxisValueToDecimal(inputVal: string): number {
+    const inputValFormat = this.getInputValueFormat(inputVal)
+
+    switch (inputValFormat) {
+      case 'powerNotation':
+        return MathUtils.convertPowerNotationToDecimal(inputVal)
+      case 'scientificNotation':
+        return MathUtils.convertScientificNotationToDecimal(inputVal)
+      case 'decimal':
+        return parseFloat(inputVal)
+      case 'invalidFormat':
+        return NaN
+      default:
+        return NaN
+    }
+  }
+
+  setInputValue(axisSetId: number, axisKey: AxisKey, inputVal: string): void {
+    if (!this.inputValues[axisSetId]) {
+      this.inputValues[axisSetId] = {
+        x1: '0',
+        x2: '0',
+        y1: '0',
+        y2: '0',
+      }
+    }
+
+    this.inputValues[axisSetId][axisKey] = inputVal
+  }
+
+  setConvertedAxisValToAxisSet(axisSetId: number, axisKey: AxisKey): void {
+    const inputAxisVal = this.getAxisSetInputValues(axisSetId)[axisKey]
+    const convertedAxisValue = this.convertAxisValueToDecimal(inputAxisVal)
+    this.axisSetRepository.activeAxisSet[axisKey].value = convertedAxisValue
+  }
+
+  canInputValueMultipliedAndDividedByTen(inputVal: string): boolean {
+    const inputValFormat = this.getInputValueFormat(inputVal)
+
+    if (inputValFormat === 'decimal') {
+      return true
+    }
+
+    if (inputValFormat === 'scientificNotation' && inputVal.startsWith('1e')) {
+      return true
+    }
+
+    return false
+  }
+
+  private getInputValueMultipliedByTen(
```

Comment:
The methods getInputValueMultipliedByTen and getInputValueDividedByTen have quite a bit of code duplication, so it seems like it could be reduced.